components/rich_text_editor/extensions/emoji/EmojiList.vue

```diff
@@ -0,0 +1,118 @@
+<template>
+  <div class="d-p4 d-bar8 d-bgc-contrast d-fs-200 d-truncate">
```

Comment:
same thing here with the utility classes.

Reply:
done moved it.